recipes-multimedia/gstreamer/gstreamer1.0_%.bbappend

```diff
@@ -1,4 +1,5 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+FILESEXTRAPATHS_prepend_mx8 := "${THISDIR}/${PN}_${PV}:"
```

Comment:
@arshadaleem66 LGTM, we can add a comment like "to manage two different GStreamer versions" also `_qemuall` override will make code self-explanatory that this is a QEMU specific implementation not for a specific QEMU architecture (qemuarm, qemux86) or any other.

Reply:
@sajjad238 I've added a comment specific to imx8 only.
I've committed this change with keeping gst version of oe-core as default and updating it only in case of imx8 bsp.
That way, we don't need to add qemu override because that assignment is being done in all cases.

Reply:
@arshadaleem66 means anything other than the FSL platform will work with this changeset, **looks good**.